### ISSUE TITLE
Possible regression: `unknown request type`

### ISSUE THREAD

**Issue by danielyli** [ktr0731/evans#55]
I recently upgraded Evans from 0.2.8 to 0.3.1, and I started experiencing this error upon launching Evans:

```
$ evans outer/inner/MyService.proto
outer/inner/MyService.proto:31:15: method outer.inner.MyService.SomeMethod: unknown request type SomeMethodRequest
```

`SomeMethodNameRequest` is defined in `outer/inner/MyService.proto` as:

```
package outer.inner;
import "outer/messages.proto";

service MyService {
  rpc SomeMethod (SomeMethodRequest) returns (SomeMethodResponse);
}
```

`SomeMethodRequest` is defined in `outer/messages.proto` as:

```
package outer;

message SomeMethodRequest {}
message SomeMethodResponse {}
```

I suspect Evans is having trouble importing a message from an outer package.

**Comment by danielyli** [comment]
After reverting to 0.2.8, I still get the same message, so I don’t know why it worked for me before.

Regardless, `protoc` compiles the definitions, so I can confirm it’s not a protos file error.

**Comment by ktr0731** [comment]
thanks for reporting a bug!
I fixed in #56. please update to 0.3.2.

**Comment by danielyli** [comment]
Such a quick bug fix! Another reason why I like this tool. :)
